### PR TITLE
Add sponsors and meta::hack page

### PR DESCRIPTION
**Pull request by tsibley**
Not done yet as I'm waiting on the following:

- [x] Final list of sponsor logos and names for the front page
- [x] Final list of metallic sponsorship levels for the `/about/meta_hack` page
- [x] Confirmation of sponsor ¶s for Booking.com and Elastic
- [x] Sponsor ¶ for Easyname
- [x] The specific order, if any, that should be used for sponsors within metallic levels

Once I know the final sponsor levels, I can:

- [x] Add platinum sponsors' logos above the search box on the front page

As more folks finalize blog entries, I can add them to the links.  The ones I've seen drafts of are:

- [x] Doug
- [ ] Olaf
- [x] Leo
- [ ] Joel

### CODE REVIEW

**Comment by tsibley** [comment]
Waiting on @neilb for some information, but if others could take a look at this locally and make sure it looks reasonable for them right now, that'd be useful.

File: root/about/meta_hack.html [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+
+    <div class="row">
+      <div class="col-md-12">
+        <h2 class="text-center">Silver Sponsors</h2>
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="col-sm-2 col-xs-12">
+        <a href="https://opensource.careers/perl-careers/">
```
Comment (pjlsergeant) [inline root/about/meta_hack.html]: While it (currently) redirects to the same place, please link to http://perl.careers/ instead everywhere you currently have this URL

**Comment by ranguard** [comment]
@tsibley my blog is live: http://blogs.perl.org/users/leo_lapworth/2016/11/metahack---metacpan-upgrade.html

**Comment by tsibley** [comment]
I believe this is ready for merge (modulo pending Travis results, I suppose).

The branch makes the front page look like this:

image

**Comment by neilb** [comment]
:+1: looks good @tsibley!
Can you set the alt text on the Perl Careers image to be:

> Find Perl Jobs and Perl Developers with Perl Careers

Thanks!

**Comment by tsibley** [comment]
@neilb Done.

**Comment by neilb** [comment]
Thanks @tsibley!

**Pull request merged**